floris/tools/floris_interface_legacy_reader.py

```diff
@@ -0,0 +1,184 @@
+# Copyright 2021 NREL
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+# See https://floris.readthedocs.io for documentation
+
+
+from __future__ import annotations
+
+import copy
+from pathlib import Path
+
+from floris.tools import FlorisInterface
+
+
+class FlorisInterfaceLegacyV2(FlorisInterface):
```

Comment:
I could see this file also including a `if __name__=="__main__"` so that it could be executed directly to convert a file from v2 to v3 without running the simulation.